src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -528,6 +529,8 @@ def _forward(self, model_inputs, return_timestamps=False, **generate_kwargs):
                 if return_timestamps == "word":
                     generate_kwargs["return_token_timestamps"] = True
                     generate_kwargs["return_segments"] = True
+                    if return_language:
+                        generate_kwargs["keep_special_tokens"] = 2
```

Comment:
Could you explain the reasoning behind this hardcoded value? (So someone who stumbles on this later on knows why this was done)